Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,11 @@
-function sum(elements) {
+function findMax(arr) {
```

Comment:
Have you tested this function yet? How do you know if it is working correctly?

Reply:
thanks for your review CJ, I made a mistake in this file, then I tried to fix and updated it.